LaunchDarkly/LaunchDarkly/LDClient.swift

```diff
@@ -753,6 +761,10 @@ public class LDClient {
     - parameter completion: Closure called when the embedded `setOnline` call completes. Takes a Bool that indicates whether the completion timedout as a parameter. (Optional)
```

Comment:
Line 760 could be a good spot to include additional info per @louis-launchdarkly 's [suggestion](https://launchdarkly.slack.com/archives/CSGD4E4LB/p1712012043905799):

```
    - parameter startWaitSeconds: A TimeInterval that determines when the completion will return if no flags have been returned from the network. If you use a large TimeInterval and wait for the timeout, then any network delays will cause your application to wait a long time before continuing execution.
```